apps/src/templates/instructions/codeReviewV2/CodeReviewTimelineReview.jsx

```diff
@@ -3,22 +3,128 @@ import PropTypes from 'prop-types';
 import CodeReviewTimelineElement, {
   codeReviewTimelineElementType
 } from '@cdo/apps/templates/instructions/codeReviewV2/CodeReviewTimelineElement';
+import FontAwesome from '@cdo/apps/templates/FontAwesome';
+import Button from '@cdo/apps/templates/Button';
+import moment from 'moment';
+import javalabMsg from '@cdo/javalab/locale';
+import Comment from '@cdo/apps/templates/instructions/codeReview/Comment';
+
+const CodeReviewTimelineReview = ({
+  review,
+  comments = [],
+  isLastElementInTimeline
+}) => {
+  const {createdAt, isClosed, projectVersion, isVersionExpired} = review;
+  const formattedDate = moment(createdAt).format('M/D/YYYY [at] h:mm A');
 
-const CodeReviewTimelineReview = ({isLastElementInTimeline}) => {
   return (
     <CodeReviewTimelineElement
       type={codeReviewTimelineElementType.CODE_REVIEW}
       isLast={isLastElementInTimeline}
+      projectVersionId={projectVersion}
+      isProjectVersionExpired={isVersionExpired}
     >
-      <div style={{backgroundColor: 'white', height: '40px'}}>
-        Code review placeholder
+      <div style={styles.wrapper}>
+        <div style={styles.header}>
+          <div style={styles.icon}>
+            <FontAwesome icon="comments-o" />
+          </div>
+          <div style={styles.title}>
+            <div style={styles.codeReviewTitle}>{javalabMsg.codeReview()}</div>
+            <div style={styles.date}>
+              {javalabMsg.openedDate({date: formattedDate})}
+            </div>
+          </div>
+          {!isClosed && (
+            <div>
+              <Button
+                icon="close"
+                style={{fontSize: 13, margin: 0}}
+                onClick={() => {}}
+                text={javalabMsg.closeReview()}
+                color={Button.ButtonColor.blue}
+              />
+            </div>
+          )}
+        </div>
+        {comments.map(comment => (
+          <Comment
+            comment={comment}
+            key={`code-review-comment-${comment.id}`}
+            onResolveStateToggle={() => {}}
+            onDelete={() => {}}
+            viewAsCodeReviewer={true}
+          />
+        ))}
+        {!isClosed && (
+          <div style={{border: '1px solid black'}}>
+            Comment editor placeholder
```

Comment:
Is this meant to be like a 'type comment here' text? I can't quite tell by the name.

Reply:
This is for a component that I haven't built yet, it'll be the comment editor with the code insert functionality
<img width="287" alt="Screen Shot 2022-05-02 at 3 25 53 PM" src="https://user-images.githubusercontent.com/24235215/166336848-9f4b8cc7-8e7d-4cd2-90bc-bb8ca5f08fff.png">

Reply:
Ohh got it. Thanks for explaining!